app/views/layouts/application.html.erb

```diff
@@ -3,7 +3,15 @@
   <head>
     <title><%= content_for(:page_title) || "Samson" %></title>
     <%= stylesheet_link_tag    "application", media: "all" %>
+    <% Samson::Hooks.plugins.map(&:stylesheets).flatten.each do |css| %>
+      <%= stylesheet_link_tag asset_path(css) %>
+    <% end %>
+
     <%= javascript_include_tag "application" %>
+    <% Samson::Hooks.plugins.map(&:javascripts).flatten.each do |javascript| %>
```

Comment:
I can't really do that since the javascripts I would have to include are now dynamic. ```assets:precompile``` happens before erb translates to clean js.

To be consistent I did the css in the same way.